app/views/site_page/static_content.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= render 'front/cover'%>
+
+<div class="c-content">
+  <div class="wrapper">
+    <%= @site_page.content['body'].html_safe unless @site_page.content['body'].blank? %>
+  </div>
+</div>
+
+<%= render 'front/footer'%>
+
+<script type="text/javascript">
+  window.route = 'OpenContent';
```

Comment:
@clementprdhomme this new page type I added in this PR is meant to be for pages like "terms and privacy", which internally will behave pretty much like the open content pages. I suggest we leave the this like this, and duplicate it later if needed